cmd/gapic-showcase/run.go

```diff
@@ -60,6 +67,31 @@ func init() {
 				grpc.StreamInterceptor(observerRegistry.StreamInterceptor),
 				grpc.UnaryInterceptor(observerRegistry.UnaryInterceptor),
 			}
+
+			// load mutual TLS cert/key and root CA cert
+			if tlsCaCert != "" || tlsCert != "" || tlsKey != "" {
+		    		x509KeyPair, err := tls.LoadX509KeyPair(tlsCert, tlsKey)
+		                if err != nil {
+	            	        	log.Fatalf("Failed to load server TLS cert/key with error:%v", err)
+		    		}
+		    		caCert, err := ioutil.ReadFile(tlsCaCert)
+		    		if err != nil {
+	            			log.Fatalf("Failed to load root CA cert file with error:%v", err)
+		    		}
+				caCertPool := x509.NewCertPool()
+				caCertPool.AppendCertsFromPEM(caCert)
+				ta := credentials.NewTLS(&tls.Config{
+					Certificates: []tls.Certificate{x509KeyPair},
+					ClientCAs:    caCertPool,
+					ClientAuth:   tls.RequireAndVerifyClientCert,
+				})
+
+				opts = []grpc.ServerOption{
+					grpc.StreamInterceptor(observerRegistry.StreamInterceptor),
+					grpc.UnaryInterceptor(observerRegistry.UnaryInterceptor),
+					grpc.Creds(ta),
+				}
```

Comment:
```suggestion
				opts = append(opts, grpc.Creds(ta))
```

This slice is already declared and initialized with the interceptors prior to this block, so we don't need to initialize it again.